cinch/roles/nginx/handlers/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: reload systemd
+  command: systemctl daemon-reload
+  when: ansible_distribution == "RedHat" and ansible_distribution_major_version == "7"
```

Comment:
Shouldn't this rely on the value of ansible_service_mgr instead of sniffing out versions in particular?